src/main/java/uk/gov/pay/products/client/publicapi/model/CardDetails.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.pay.products.client.publicapi.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
+public class CardDetails {
+
+    private final String lastDigitsCardNumber;
+    private final String cardHolderName;
+    private final String expiryDate;
+    private final Address billingAddress;
+    private final String cardBrand;
```

Comment:
I have changed that into an enum. Regarding publishing the client as a separate package, it has been discussed already, this will most likely happen later.